ui/src/pages/search.js

```diff
@@ -1,28 +1,67 @@
 import React, { useEffect, useState } from 'react';
-import GameService from '../services/gameService';
+import gameService from '../services/gameService';
+import ReactPaginate from 'react-paginate';
+import history from '../history';
 
 import '../styles/search.css';
 import SearchResult from '../components/SearchResult';
 
 const SearchPage = ({ searchTerm, pageNum }) => {
   const [gameComponents, setGameComponents] = useState('');
+  const [numPages, setNumPages] = useState('');
 
   useEffect(() => {
     const updateGameComponents = async () => {
-      const res = await GameService.gameSearch({
+      const res = await gameService.gameSearch({
         searchTerm: searchTerm,
         pageNum: pageNum,
       });
       if (res.status === 200) {
         setGameComponents(res.body.map((game) => <SearchResult key={game.id} {...game} />));
       }
     };
+
+    const getTotalNumberOfPages = async () => {
+      const res = await gameService.getSearchInfo({
+        searchTerm: searchTerm,
+      });
+
+      if (res.status === 200) {
+        setNumPages(res.body.maxNumPages + 1);
+      }
+    };
+
     updateGameComponents();
+    getTotalNumberOfPages();
   }, [searchTerm, pageNum]);
```

Comment:
Not sure if we need to bind the searchInfo API call to refresh every time the pageNum changes. It should probably be in its own useEffect hook.